### ISSUE TITLE
Crash when reforging an unregistered curios type

### ISSUE THREAD

**Issue by medsal15** [Daripher/Apothic-Curios#9]
No idea whether this is a duplicate of #5 or a separate issue, but my game crashed when I tried to reforge Greed Of Nidhoggur from [L2 Hostility](https://www.curseforge.com/minecraft/mc-mods/l2hostility)

The error is `java.lang.RuntimeException: Failed to locate any affixes for greed_of_nidhoggur{null} with category LootCategory[curios:hostility_curse] and rarity LootRarity{apotheosis:mythic}.`

**Comment by SiverDX** [comment]
@Daripher personally I'm doing this

```java
@Mixin(value = LootController.class, remap = false)
public abstract class MixinLootController {
    @Inject(method = "createLootItem(Lnet/minecraft/world/item/ItemStack;Lshadows/apotheosis/adventure/loot/LootCategory;Lshadows/apotheosis/adventure/loot/LootRarity;Lnet/minecraft/util/RandomSource;)Lnet/minecraft/world/item/ItemStack;", at = @At(value = "NEW", target = "(Ljava/lang/String;)Ljava/lang/RuntimeException;", shift = At.Shift.BEFORE), cancellable = true)
    private static void stopException(final ItemStack stack, final LootCategory category, final LootRarity rarity, final RandomSource random, final CallbackInfoReturnable<ItemStack> callback) {
        callback.setReturnValue(stack);
        TrialAndError.LOG.error("Failed to locate any affixes for [{} | {}] with category {} and rarity {}.", stack.getItem(), stack.getTag(), category, rarity);
    }
}
```

**Comment by medsal15** [comment]
Fixed in 1.0.3b which no longer allows unreforgable curios in the reforging table
